pages/common/zig.md

```diff
@@ -0,0 +1,36 @@
+# zig
+
+> The Zig compiler and toolchain.
+> More information: <https://ziglang.org/>.
+
+- Compile the project in the current directory:
+
+`zig build`
+
+- Compile and run the project in the current directory:
+
+`zig build run`
+
+- Initialize a `zig build` application:
+
+`zig init-exe`
+
+- Intitialize a `zig build` library:
+
+`zig init-lib`
+
+- Create and run a test build:
+
+`zig test {{path/to/file.zig}}`
+
+- Reformat Zig source into canonical form:
```

Comment:
Why not `format`?
```suggestion
- Format Zig source into canonical form:
```

Reply:
Because this is how Zig's help words it:
```shell
$ zig --help | awk '/fmt/ { print $0 }'
  fmt              Reformat Zig source into canonical form
```

Reply:
Are you like too fancy to use grep?

Reply:
I've been using Awk so much recently, I've forgotten about grep it seems. 👀